docs/configuration/configuration.md

```diff
@@ -490,6 +490,73 @@ instances.
 
 One of the following `<openstack_role>` types can be configured to discover targets:
 
+### `<cvm_sd_config>`
+
+CVM SD configurations allow retrieving scrape targets from TencentCloud CVM
+instances. The private IP address is used by default, but may be changed to
+the public IP address with relabeling.
+
+The following meta labels are available on targets during [relabeling](#relabel_config):
+
+
+* `__meta_cvm_project_id`: the project id of the CVM instance
+* `__meta_cvm_region`: the region of the CVM instance		
+* `__meta_cvm_host_id`: the host id of the CVM instance's host machine, if available
+* `__meta_cvm_zone`: the availability zone in which the CVM instance is running
+* `__meta_cvm_instance_id`: the CVM instance ID
+* `__meta_cvm_instance_state`: the state of the CVM instance
+* `__meta_cvm_instance_type`: the type of the CVM instance
+* `__meta_cvm_private_ip`: the private IP address of the CVM instance, if present
+* `__meta_cvm_public_ip`: the public IP address of the CVM instance, if available
+* `__meta_cvm_vpc_id`: the ID of the VPC in which the CVM instance is running, if available
+* `__meta_cvm_subnet_id`: the subnet ID of the network interface
+* `__meta_cvm_os_name`: the Operating System of the CVM instance
+* `__meta_cvm_tag_<tagkey>`: each tag value of the CVM instance
+
+
+See below for the configuration options for CVM discovery:
+
+```yaml
+# The information to access the CVM API.
+
+# The TencentCloud region. 
+[ region: <string> ]
+
+# Custom endpoint to be used.
+[ endpoint: <string> ]
+
+# The TencentCloud API keys. If blank, the environment variables `PROMETHEUS_DS_CVM_SECRET_ID`
+# `PROMETHEUS_DS_CVM_SECRET_KEY` and `PROMETHEUS_DS_CVM_TOKEN` are used.
+[ secret_id: <string> ]
+[ secret_key: <secret> ]
```

Comment:
are the 3 of them actually required?
can we read the token from a file?

Reply:
token is optional for these who has only has temporary secretid/ secrettoken, comment is added for configuration.md